AdyenDropIn/DropInComponent.swift

```diff
@@ -159,8 +159,9 @@ public final class DropInComponent: NSObject, PresentableComponent {
         }
     }
     
-    private func didSelectCancelButton(isRoot: Bool, component: PaymentComponent?) {
+    private func didSelectCancelButton(isRoot: Bool, component: PresentableComponent) {
         guard !paymentInProgress else { return }
+        component.didCancel()
```

Comment:
This will do nothing, righ?
Default implementation is blank...

Reply:
`PollingAwaitComponent` use it to stop polling.